packages/wattpm/lib/commands/logs.js

```diff
@@ -0,0 +1,83 @@
+import { RuntimeApiClient } from '@platformatic/control'
+import { ensureLoggableError } from '@platformatic/utils'
+import pinoPretty from 'pino-pretty'
+import split2 from 'split2'
+import { getMatchingRuntimeArgs, parseArgs } from '../utils.js'
+
+export async function logsCommand (logger, args) {
```

Comment:
Why have you reimplemented this? Everything was already present in https://github.com/platformatic/platformatic/blob/main/packages/control/control.js#L25-L34.

Reply:
I misunderstood the scope of the issue. Sorry for that.